CHANGELOG.md

```diff
@@ -17,6 +17,9 @@
 * [#6006](https://github.com/bbatsov/rubocop/pull/6006): Remove `rake repl` task. ([@koic][])
 
 ## 0.57.2 (2018-06-12)
+### Changes
+
+* Drop support for MRI 2.1. ([@drenmi][])
```

Comment:
Feel free to reorder this. I don't have a strong preference on the subject, but ideally we should use a consistent ordering in the changelog.

Reply:
My comment was wrong 💦 
Since this item has not been released yet, it has moved to `master (unreleased)` section with the following commit.
https://github.com/rubocop-hq/rubocop/commit/5f70faeeb15f768a09a68d3921950b6b382a3705

Reply:
Build is failing because of a missing space. Ideally let's not push to `master`. 😬

Reply:
I'm sorry about that. I was not careful in pushing to master branch 😥 

I confirmed that it was fixed by the following change.
https://github.com/rubocop-hq/rubocop/commit/a073d9af17ccca7e8d483b787a13611c8511b373#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR18

As I changed changelog document only, I forgot to run `bundle exec rake`. The push to master branch should be confirmed more properly locally than open a PR will checked with CI.

I thought that push to master branch was preferable, as far as minor changes (e.g. fix typo, cosmetic change) were concerned, rather than being notified to the watching users by opening a PR. Of course the contents to be reviewed opens a PR. (So, I will open a PR as a basis.)

If pushing to master branch, I pay close attention so that builds can be more securely passed.

Reply:
Don't worry, @koic. Mistakes happen to everyone. I'm fine with people fixing small things directly on `master`, I just want everyone to be very careful when doing this.